docs/preview/03-Features/powershell/azure-logic-apps-standard.md

```diff
@@ -0,0 +1,354 @@
+---
+title: "Azure Logic Apps - Standard"
+layout: default
+---
+
+# Azure Logic Apps - Standard
```

Comment:
Just to check: this file is almost a copy/paste of the other one, except that it allows for additional parameters?

Reply:
It is mostly the same indeed, but other parameters are required or not relevant.
I split them up between consumption and standard because otherwise the docs became unclear (a lot of text, multiple examples, etc)